python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -64,8 +64,12 @@ def xbdm_parse_response2(length=None):
 
 #FIXME: For legacy reasons, should be updated?
 def xbdm_parse_response(length=None):
-  return xbdm_parse_response2(length)[1]
-  
+
+  tempvar = xbdm_parse_response2(length) #with status code
+  tempvar = tempvar[1:len(tempvar)]      #strip status code
+  tempvar = list(tempvar)                #to list  
+  tempvar = tempvar[0]                   #"unpack" list 
+  return tempvar
```

Comment:
I'm not sure why all of this would be necessary. I'll have to do some experiments with python tuples.

Also `[1:]` would grab all elements starting at index 1 til the end (no need for the len() stuff)